swan-lake/learn/observing-ballerina-code.md

```diff
@@ -293,57 +286,53 @@ specification.](https://github.com/opentracing/specification/blob/master/semanti
 
 ### Configuring Advanced Tracing for Ballerina
 
-Tracing can be enabled in Ballerina with the `--b7a.observability.enabled=true` flag as mentioned in the [Observing a Ballerina Service](#observing-a-ballerina-service) section, as well as a configuration option. This section mainly focuses on the configuration options with the description and possible values.
+Tracing can be enabled in Ballerina with the few configurations as mentioned in the
+[Observing a Ballerina Service](#observing-a-ballerina-service) section.
+This section mainly focuses on the configuration options with the description and possible values.
 
-The sample configuration that enables tracing, and uses Jaeger as the sample tracer as provided below.
+The sample configuration that enables tracing, and uses Jaeger as the tracer as provided below.
 
 ```toml
-[b7a.observability.tracing]
-enabled=true
-name="jaeger"
+[ballerina.observe]
+tracingEnabled=true
+tracingProvider="jaeger"
 ```
 
 The table below provides the descriptions of each configuration option and possible values that can be assigned.
 
 Configuration Key | Description | Default Value | Possible Values
 --- | --- | --- | --- 
-b7a.observability.tracing.enabled | Whether tracing is enabled (true) or disabled (false) | false | true or false
-b7a.observability.tracing.provider | Tracer name which implements tracer interface. | jaeger | jaeger or if any custom implementation, the name of the tracer.
+ballerina.observe.tracingEnabled | Whether tracing is enabled (true) or disabled (false) | false | true or false
+ballerina.observe.tracingProvider | Tracer name which implements tracer interface. | jaeger | jaeger or if any custom implementation, the name of the tracer.
```

Comment:
```suggestion
ballerina.observe.tracingProvider | The tracer name, which implements the tracer interface. | jaeger | Jaeger or the name of the tracer of any custom implementation.
```

Reply:
Name of the tracer ("jaeger") should be lower case as it is a configuration. Other suggestions had been fixed in 146f6434f70aee59d33f47a274e40ed2ac703d6a

Reply:
+1